README.md

```diff
@@ -8,6 +8,21 @@ For details, see the executable documentation in the `vroom/` directory or the
 helpfiles in the `doc/` directory. The helpfiles are also available via
 `:help codefmt` if codefmt is installed (and helptags have been generated).
 
+# Supported File-types
```

Comment:
I should say, the plugin itself should have some indication about what it supports, but probably only one source of truth. Maybe a ListFormatters command. Maybe ListFormatters is sufficient. I dunno. I think it's not great as vimdoc, since external sources can register formatters.